src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/ccd/CCDConfigConstant.java

```diff
@@ -212,5 +212,18 @@
     public static final String BULK_PRINT_LETTER_ID_RES = "bulkPrintLetterIdRes";
     public static final String UPLOAD_ORDER = "uploadOrder";
 
+    //Contested Case Documents
+    public static final String CONTESTED_UPLOADED_DOCUMENTS = "uploadCaseDocument";
+    public static final String APPLICANT_CORRESPONDENCE_COLLECTION = "appCorrespondenceCollection";
+    public static final String APPLICANT_FR_FORM_COLLECTION = "appFRFormsCollection";
+    public static final String APPLICANT_EVIDENCE_COLLECTION = "appEvidenceCollection";
+    public static final String APPLICANT_TRIAL_BUNDLE_COLLECTION = "appTrialBundleCollection";
+    public static final String RESPONDENT_CORRESPONDENCE_COLLECTION = "respCorrespondenceCollection";
+    public static final String RESPONDENT_FR_FORM_COLLECTION = "respFRFormsCollection";
+    public static final String RESPONDENT_EVIDENCE_COLLECTION = "respEvidenceCollection";
+    public static final String RESPONDENT_TRIAL_BUNDLE_COLLECTION = "respTrialBundleCollection";
+
+    //public static final String RESPONDENT_CASE_DOCUMENTS = "PLACEHOLDER";
```

Comment:
?

Reply:
This was back when I thought I was going to have an applicant documents and a respondent documents collection. However after speaking with Grace and John, we need 4 seperate colections for each "Document Header"